modules/i18n/src/models/common.ts

```diff
@@ -172,6 +172,7 @@ export interface ModalInterface {
     content?: object;
     primaryButton: string;
     secondaryButton: string;
+    [ key: string ]: any;
```

Comment:
Shall we have the exact attributes here. Since the purpose of the interface is to force other languages to implement the same without missing the newly added attributes.

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1732/commits/6ce4484d6f7bfe758f7f22f590a45a499e21d508